MomentumDiscordBot/Services/SlashCommandService.cs

```diff
@@ -52,5 +62,65 @@ private Task _commands_SlashCommandErrored(SlashCommandsExtension sender, SlashC
 
             return Task.CompletedTask;
         }
+
+        private Task _discordClient_GuildsDownloaded(DiscordClient sender, GuildDownloadCompletedEventArgs e)
+        {
+
+            _ = Task.Run(async () =>
+            {
+                _textChannel = await _discordClient.GetChannelAsync(_config.AdminBotChannel);
+                await FindRestartMessageAsync();
+            });
+
+            return Task.CompletedTask;
+        }
+
+        private async Task FindRestartMessageAsync()
+        {
+            // Filter only messages from this bot
+            var existingMessages = (await _textChannel.GetMessagesAsync()).FromSelf(_discordClient);
+
+            foreach (var message in existingMessages)
+            {
+                var (result, restartMessage) = await TryFindRestartMessageAsync(message);
+                if (result)
+                {
+                    // restart message found
+                    if(restartMessage != null)
+                    {
+                        // restartMessage is null when we already responded
+
+                        var embed = new DiscordEmbedBuilder
+                        {
+                            Description = "back after " + AdminModule.ForcerestartCommandName,
```

Comment:
Could we show the time diff here? Since we have both the past message object already, we should be able to compare it to the current time with 
```
var diff = DateTimeOffset.Now - restartMessage.Timestamp;
```
then set the Description to 
```
$"Restart complete! Took {diff.TotalSeconds} seconds."
```

I tried this out just now and seems to work.